10-cmdline.md

```diff
@@ -610,3 +610,10 @@ the program now does everything we set out to do.
 >
 > *   If no filenames are given, it reports the number of lines in standard input.
 > *   If one or more filenames are given, it reports the number of lines in each, followed by the total number of lines.
+
+> ## Generate an Error message {.challenge}
+>
+> Write a program called `check_argument.py` that print usage then exit program if there are no arguments are submitted. (Hint) You can use `sys.exit()` for exit program.
```

Comment:
Also, to echo what my comment from #184:

Could you add some line breaks here? We currently don't do this very consistently, but in most of our `.md` files, we have newlines either after a certain number of characters, or when it makes sense for the sentence (see [this blog post](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) for details). The main reason for this is so that if someone makes a change to this paragraph later on, it will be more obvious what has changed because diffs are done on a line-by-line basis.